config/env.js

```diff
@@ -46,48 +44,56 @@ dotenvFiles.forEach(dotenvFile => {
 // It works similar to `NODE_PATH` in Node itself:
 // https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders
 // Note that unlike in Node, only *relative* paths from `NODE_PATH` are honored.
-// Otherwise, we risk importing Node.js core modules into an app instead of Webpack shims.
```

Comment:
ha, seems like even CRA team was writing it wrong :)

Reply:
![image](https://user-images.githubusercontent.com/1477453/82491752-be5e5000-9aed-11ea-83d0-8fed803db49f.png)